packages/components/slider/src/use-slider-thumb.ts

```diff
@@ -101,16 +109,19 @@ export function useSliderThumb(props: UseSliderThumbProps) {
   };
 
   const getTooltipProps = () => {
-    const value = state.values[index ?? 0];
+    const value = numberFormatter
+      ? numberFormatter.format(state.values[index ?? 0])
+      : state.values[index ?? 0];
 
     const content =
       renderOutput && typeof renderOutput === "function" ? renderOutput(`${value}`) : value;
 
     return {
       ...tooltipProps,
-      content,
-      isOpen: isHovered || isDragging,
-      placement: isVertical ? "right" : "top",
+      placement: tooltipProps?.placement ?? isVertical ? "right" : "top",
+      content: tooltipProps?.content ?? content,
+      updatePositionDeps: [isDragging, isHovered, value],
+      isOpen: tooltipProps?.isOpen ?? (isHovered || isDragging),
     } as TooltipProps;
```

Comment:
Can you remove this?
```suggestion
    } as TooltipProps;
```

Reply:
Nop, we need explicit returns

![CleanShot 2023-10-07 at 16 22 58](https://github.com/nextui-org/nextui/assets/30373425/853ddce0-fe04-45aa-a76e-09e90dc8a17d)